client/components/TestPlanVersionsPage/index.jsx

```diff
@@ -409,18 +420,19 @@ const TestPlanVersionsPage = () => {
                     ).length;
                 return (
                     <div key={testPlanVersion.id}>
-                        <H3>
+                        <H2>
                             <VersionString
                                 date={testPlanVersion.updatedAt}
                                 iconColor={getIconColor(testPlanVersion)}
                                 fullWidth={false}
                                 autoWidth={false}
                             />
+                            &nbsp;
```

Comment:
I think this is just a problem between different screenreaders but on VO this outputs:

> heading level 2 5 items V22.09.28  **space** Deprecated  on  Oct 24, 2022

If I remove this `&nbsp;` the "space" is no longer read but I assume this is necessary to prevent the running together issue on other screenreaders (which doesn't happen with VO on the target branch). Just wanted to add a note.

Reply:
Very odd. Double checked on NVDA and can confirm it comes out pressed against each other without adding in the `nbsp` like so:

> heading    level 2  V22.10.24R&D on Oct 24, 2022

VO then reads it properly though like that. So pretty puzzling there. Going to spend some time looking for a better approach before merging. Thanks for the review!

Reply:
@stalgiag it seems to be a known issue with VO so I've gone ahead and used an aria-label on the `h2` instead

Reply:
Exactly the kind of problem this project will hopefully prevent in the future. Thanks for looking into it!